pages/clubs/Running.jsx

```diff
@@ -9,39 +11,28 @@ var Running = React.createClass({
   contextTypes: {
     intl: React.PropTypes.object
   },
-  setData : function(data){
-    for(var i = 0; i < data.length; i++){
-      var categoryLabel = data[i].category.toLowerCase();
+  componentDidMount: function() {
+    this.props.teachAPI.getClubsGuides(function(err, res){
```

Comment:
protip: make this a component function, so `this.props.teachAPI.getClubsGuides(this.processClubsGuides)`